src/components/scrollbars/CustomScrollContainer.tsx

```diff
@@ -14,9 +14,8 @@ const ScrollTrack: ScrollbarsProps["renderTrackHorizontal"] = props => {
 
 export const CustomScrollContainer: FC<ScrollbarsProps> = props => (
   <Scrollbars
-    autoHeight
-    autoHeightMax={Number.MAX_SAFE_INTEGER}
     {...props}
+    autoHeightMax={Number.MAX_SAFE_INTEGER}
```

Comment:
this way, you won't be able to override this value by passing the props

Reply:
Will be adjusted in next pr